tests/rt.sh

```diff
@@ -416,14 +444,14 @@ if [[ $TESTS_FILE =~ '35d' ]]; then
 fi
 
 BL_DATE=20210907
-if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID = cheyenne.* ]] || [[ $MACHINE_ID = gaea.* ]] || [[ $MACHINE_ID = jet.* ]]; then
+if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID = cheyenne.* ]] || [[ $MACHINE_ID = gaea.* ]] || [[ $MACHINE_ID = jet.* ]] || [[ $MACHINE_ID = s4.* ]]; then
   RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-${BL_DATE}/${RT_COMPILER^^}}
 else
   RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-${BL_DATE}}
 fi
 
 INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20210825}
-INPUTDATA_ROOT_WW3=${INPUTDATA_ROOT}/WW3_input_data_20210621
+INPUTDATA_ROOT_WW3=${INPUTDATA_ROOT}/WW3_input_data_20210908
```

Comment:
Why is this being updated? Do we have the updated data set on the other platforms? What is different to the 20210621 input data?

Reply:
This is the dataset that is being used to test #790, which this PR is reliant on.  That data has been added to at least Hera, Gaea, Orion, and WCOSS-Dell.  I will defer to @aliabdolali regarding the difference in the data.

Reply:
These are updated mod_def files for waves for the updated code.

Reply:
Ok, thanks. We will need to copy these data over to Jet and Cheyenne.